src/modules/commander/commander_params.c

```diff
@@ -907,3 +907,18 @@ PARAM_DEFINE_INT32(COM_ASPD_FS_ACT, 0);
  */
 PARAM_DEFINE_INT32(COM_ASPD_FS_DLY, 0);
 
+/**
+ * User Flight Profile
+ *
+ * Describes the intended use of the vehicle.
+ * Can be used by ground control software or log post processing.
+ * This param does not influence the behavior within the firmware. This means for example the control logic is independent of the setting of this param (but depends on other params).
+ *
+ * @value 0 Default
+ * @value 100 Pro User
+ * @value 200 Flight Tester
+ * @value 300 Developer
+ *
```

Comment:
Extra blank line :smiley:

Reply:
I found that more readable and I see that for some of the other params in the file there is also a blank line and for others there is not. I am not opinionated. What is the standard here?